src/utils/api/sumstats.helper.ts

```diff
@@ -0,0 +1,39 @@
+import { OverallStats } from "types/overallstats.trpc";
+import { readAllChartdata } from "utils/database/chartdata.db";
+
+export default async function getOverallStats(): Promise<OverallStats | null> {
+  const data = await readAllChartdata();
+  if (!data) return null;
+  const rounds = data.length;
+  const bribedVotes = data.reduce((sum, a) => sum + a.bribedVotes, 0);
+  const totalVotes = data.reduce((sum, a) => sum + a.totalVotes, 0);
+  const bribedVotesRatio = Number(
+    ((bribedVotes / totalVotes) * 100).toFixed(2)
+  );
+  const totalVoter = data.reduce((sum, a) => sum + a.totalVoter, 0);
+  const totalBribes = data.reduce((sum, a) => sum + a.totalBribes, 0);
+  const totalOffers = data.reduce((sum, a) => sum + a.totalBriber, 0);
+  const avgPer1000 = Number(((totalBribes / bribedVotes) * 1000).toFixed(2));
+  const sumPriceBeets = data.reduce((sum, a) => sum + a.priceBeets, 0);
+  const avgPriceBeets = Number((sumPriceBeets / rounds).toFixed(4));
+  const votingApr = data.map((round) => {
+    const x =
+      (round.totalBribes / round.priceFbeets / round.bribedVotes) * 2600;
```

Comment:
What is the 2600? maybe a comment of it's suppose? I'm sure as soon as you tell me, it will be obviouse

Reply:
Yes, you are right, it's not very obvious: the *2600 really consists of a *100 (for percent value) and a *26 (single bribe is for 2 weeks, so *26 is for yearly, as we have 26 rounds of 2 weeks each per year)